RELEASE.md

```diff
@@ -6,12 +6,14 @@
 ## Bug fixes and other changes
 * Moved `_find_run_command()` and `_find_run_command_in_plugins()` from `__main__.py` in the project template to the framework itself.
 * Fixed a bug where `%load_node` breaks with multi-lines import statements.
+* Fixed a bug where `OmegaConfigLoader` breaks when using different keys but same high-level key.
```

Comment:
This wasn't a bug but a very deliberate feature clearly described in https://docs.kedro.org/en/stable/configuration/configuration_basics.html. I do see a good use case to change the behaviour for parameters, so let's tweak the implementation and release notes to reflect that.

Reply:
Awesome. I guess I need to update the supporting document to also mention this change right?

Reply:
Yes it would be great if you could update the docs as well!

Reply:
I updated the doc which you shared. Please let me know in case some other documentation needs updation too.